src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergInitServiceIT.java

```diff
@@ -0,0 +1,74 @@
+package com.snowflake.kafka.connector.streaming.iceberg;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.snowflake.kafka.connector.internal.SnowflakeKafkaConnectorException;
+import com.snowflake.kafka.connector.internal.TestUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class IcebergInitServiceIT extends BaseIcebergIT {
+
+  private static IcebergInitService icebergInitService;
+
+  private String tableName;
+
+  @BeforeAll
+  // overrides the base class @BeforeAll
+  public static void setup() {
+    conn = TestUtils.getConnectionServiceForStreaming();
+    icebergInitService = new IcebergInitService(conn);
+  }
+
+  @BeforeEach
+  public void setUp() {
+    tableName = TestUtils.randomTableName();
+  }
+
+  @AfterEach
+  public void tearDown() {
+    dropIcebergTable(tableName);
+  }
+
+  @Test
+  void shouldInitializeMetadataType() {
```

Comment:
And here we create iceberg table with "record_metadata" column and then run a query to alter it with "record_metadata". How doeas it work?